index.html

```diff
@@ -45,6 +39,8 @@ <h1>What would you like to accomplish during this time?</h1>
             </div>
             <button class="activity" id="start-activity" type="button">START ACTIVITY</button>
         </form>
+        <div class="countdown"><span class="minutes-left">0</span><span>:</span><span class="seconds-left">00</span></div>
```

Comment:
Nice addition of the timer div and class for the countdown timer. This will be helpful for iteration 2-3